flutter_local_notifications/docs/usage.md

```diff
@@ -0,0 +1,269 @@
+
+For more information, see the platform-specific usage guides. [This page](https://web.dev/articles/permissions-best-practices) also serves some good advice on permissions best practices.
+
+### Show a notification
+
+The [`show()`](https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/FlutterLocalNotificationsPlugin/show.html) function shows a notification immediately, and accepts five arguments:
+
+- the `id` of the notification, as an integer. This ID is used to uniquely identify the notification to the device. If you send a new notification with the same ID, it will be updated in-place, which is useful in cases like chat threads and loading notifications.
+- an optional `title` of the notification, as a string. If null, platforms will usually show the name of the app instead.
+- an optional `body` of the notification, as a string
+- an optional `NotificationDetails`, as discussed above
+- an optional notification `payload`, as a string. This will be available to your app, via `NotificationResponse.payload`, if the notification is pressed. Use this to provide enough context for your app to meaningfully respond, like taking your user to a specific page or sending a message.
+
+These same parameters will be present on most methods that show notifications. Other platform-specific functionality can be specified in the `NotificationDetails`. For an example, see [above](#the-different-details-classes)
+
+### Schedule a notification
+
+> [!NOTE]
+>
+> Not supported on Linux or the Web
+
+The [`zonedSchedule()`](https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/FlutterLocalNotificationsPlugin/zonedSchedule.html) method accepts the same five arguments as `show()`, but also accepts:
+
+- the `scheduledDate` to show the notification
+- the `uiLocalNotificationDateInterpretation`, which describes how devices prior to iOS 10 should interpret the `scheduledDate`
+- the `androidScheduleMode`, which determines how precisely the device should schedule the notification
+  - More precision requires special permissions -- see the Android setup and usage guides
+
+- an optional `matchDateTimeComponents`, which tells the OS whether and how to periodically schedule this notification
+
+Note that the `scheduledDate` is not a regular `DateTime`, but rather a [`TZDateTime`](https://pub.dev/documentation/timezone/latest/timezone.standalone/TZDateTime-class.html) from `package:timezone` to ensure location data is incorporated into the notification delivery time. See above for how to set that up and get the user's current location. For example:
```

Comment:
Rather than only saying "above", I'd recommend adding link to the section on getting timezone location. Avoids readers having to "guess" how far above they have to go